tests/algorithms/test_preference_comparisons.py

```diff
@@ -54,13 +56,18 @@ def agent(venv):
 
 
 @pytest.fixture
-def random_fragmenter():
-    return preference_comparisons.RandomFragmenter(seed=0, warning_threshold=0)
+def random_fragmenter(random_state_fixed):
+    random_state = random_state_fixed
+    return preference_comparisons.RandomFragmenter(
+        random_state=random.Random(util.make_seeds(random_state)),
```

Comment:
Why not just pass this directly in on line 62?

If that makes line too long then could split it up as `seeds = util.make_seeds(random_state)`.

Otherwise these assignments just seem vacuous.